code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1507,15 +1509,24 @@
 	icon_state = "bugburger"
 	food_flags = FOOD_MEAT
 	base_crumb_chance = 20
-
+	
 /obj/item/weapon/reagent_containers/food/snacks/gunkburger/New()
 	..()
+	processing_objects += src
```

Comment:
please remember to remove these from processing_objects in Destroy()

Reply:
The sea devils had no destroy though. 
Is it just 
```
/obj/item/weapon/reagent_containers/food/snacks/Destroy()
	processing_objects -= src
```
?

Reply:
yes, but very important, remember you MUST call ..() at the end of any Destroy() override; if you forget it will be impossible to delete the object (for example, when you finish eating it, it won't go away)

Reply:
```
/obj/item/weapon/reagent_containers/food/snacks/Destroy()
	processing_objects -= src
	 ..() 
```
Like this?

Reply:
yes that would be fine

Reply:
done.